tests/infra_test.go

```diff
@@ -1268,9 +1268,8 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 				for key := range node.Labels {
 					if strings.Contains(key, v1.CPUModelLabel) {
 						model := strings.TrimPrefix(key, v1.CPUModelLabel)
-						for obsoleteModel := range nodelabellerutil.DefaultObsoleteCPUModels {
-							Expect(model == obsoleteModel).To(Equal(false), "Node can't contain label with cpu model, which is in default obsolete filter")
-						}
+						Expect(nodelabellerutil.DefaultObsoleteCPUModels).ToNot(HaveKey(model),
+							"Node can't contain label with cpu model, which is in default obsolete filter")
```

Comment:
This change is different from the others, and is not documented in the commit message / PR description.
Do you mind doing it in a different PR?

Reply:
It's not different. I find it by searching `Equal(false)`, and the purpose is to get rid of this pattern. But just replacing it with `BeFalse()` in this case was wrong too, because this code could be simplified.

Reply:
I share your opinion that this change simplifies and improves the code, but it is not a simple replacement as depicted in the PR title and description.
IMO this change should get (at least) a commit of its own with a commit message which describes it:
> just replacing it with BeFalse() in this case was wrong too, because this code could be simplified.

Reply:
@orelmisan - thanks. I guess it's a semantic discussion... Anyway, I split the original commit to two commits, as you offered.